src/main/java/com/iota/iri/IRI.java

```diff
@@ -330,7 +331,20 @@ private static boolean isValidated(final Configuration configuration, final Stri
             if (vmaxPeers != null) {
                 configuration.put(DefaultConfSettings.MAX_PEERS, vmaxPeers);
             }
+
+            final Integer belowMaxDepthLimit = parser.getOptionValue(belowMaxDepthTxLimit);
+            if (belowMaxDepthLimit != null) {
+                configuration.put(DefaultConfSettings.BELOW_MAX_DEPTH_TRANSACTION_LIMIT,
+                        String.valueOf(belowMaxDepthLimit));
+            }
+
+            final Integer walkValidatorCache = parser.getOptionValue(walkValidatorCacheSize);
```

Comment:
maybe rename to "walkValidatorCacheSize" to be consistent with the naming in the config and Iota.java - could otherwise be mistaken with an instance of the cache (not really :) but you get the point)

Reply:
The problem is that we have 2 variables with the same name...
It will be resolved when we do #724 .